Sources/RegexBenchmark/Suite/ReluctantQuant.swift

```diff
@@ -0,0 +1,42 @@
+import _StringProcessing
+import RegexBuilder
+
+extension BenchmarkRunner {
+  mutating func addReluctantQuant() {
+    let size = 500000
+    let s = String(repeating: "a", count: size)
```

Comment:
Future work: These are very, very micro (nano?) and would be trivialized through some analysis and optimizations that wouldn't necessarily improve the state of real regexes or components of real regexes. I wonder if we should distinguish these from more realistic ones.